ScreenControl_Unity/Assets/ScreenControl/Core/Prefabs/ScreenControlCore.prefab

```diff
@@ -170,6 +170,21 @@ PrefabInstance:
       propertyPath: onValueChanged.m_PersistentCalls.m_Calls.Array.data[0].m_CallState
       value: 2
       objectReference: {fileID: 0}
+    - target: {fileID: 779857710308832473, guid: dc0e884672fe42e49b02672301208def,
```

Comment:
This may be a result of not applying changes to the prefab so maybe have another look over it?